### PR TITLE
Kubernetes drainer refactoring

### PR DESCRIPTION
**Pull request by ilkinmammadzada**
### Description

Refactoring kubernetes drainer, adding new test cases

### CODE REVIEW

File: clusterman/draining/queue.py [hunk 1]
```diff
@@ -51,6 +51,7 @@
 
 logger = colorlog.getLogger(__name__)
 DRAIN_CACHE_SECONDS = 1800
+FORCE_TERMINATION = False
```
Comment (piax93) [inline clusterman/draining/queue.py]: nit, I'd call it `DEFAULT_FORCE_TERMINATION`

File: clusterman/draining/queue.py [hunk 2]
```diff
@@ -70,6 +71,7 @@ class DrainingClient:
     def __init__(self, cluster_name: str) -> None:
         self.client = sqs
         self.cluster = cluster_name
+        self.drain_reprocessing_timeout_seconds = staticconf.read_int("drain_reprocessing_timeout_seconds", default=15)
```
Comment (piax93) [inline clusterman/draining/queue.py]: I'd add a global "constant" for this default value as well.

File: clusterman/draining/queue.py [hunk 3]
```diff
@@ -109,9 +111,10 @@ def submit_instance_for_draining(
             ),
         )
 
-    def submit_host_for_draining(self, host: Host) -> None:
+    def submit_host_for_draining(self, host: Host, delay: int) -> None:
```
Comment (piax93) [inline clusterman/draining/queue.py]: I'd set a default None/0 value, so that you can avoid updating upstream calls (e.g. line 390)

**Review by piax93 — COMMENTED** [review]
A bunch of nitpicking, LGTM otherwise

**Comment by ilkinmammadzada** [comment]
> A bunch of nitpicking, LGTM otherwise

@piax93 thanks for feedback. all done!

**Review by piax93 — APPROVED** [review]

**Pull request merged**